From 3e6eb236dd2caf00ee29beb1c68349f0f99c1fde Mon Sep 17 00:00:00 2001
From: John Kirkham <kirkhamj@janelia.hhmi.org>
Date: Tue, 6 Mar 2018 00:18:20 -0500
Subject: [PATCH 2/2] Drop test_curried_doctests

This shouldn't be include in the Python package as noted upstream. So
just go ahead and drop it.

xref: https://github.com/pytoolz/cytoolz/issues/116
---
 cytoolz/tests/test_curried_doctests.py | 11 -----------
 1 file changed, 11 deletions(-)
 delete mode 100644 cytoolz/tests/test_curried_doctests.py

diff --git cytoolz/tests/test_curried_doctests.py cytoolz/tests/test_curried_doctests.py
deleted file mode 100644
index e22efa5..0000000
--- cytoolz/tests/test_curried_doctests.py
+++ /dev/null
@@ -1,11 +0,0 @@
-import doctest
-import cytoolz
-
-
-def test_doctests():
-    cytoolz.__test__ = {}
-    for name, func in vars(cytoolz).items():
-        if isinstance(func, cytoolz.curry):
-            cytoolz.__test__[name] = func.func
-    assert doctest.testmod(cytoolz).failed == 0
-    del cytoolz.__test__
-- 
2.15.1

